### ISSUE TITLE
GH actions: macos build failing intermitently

### ISSUE THREAD

**Issue by kinow** [cylc/cylc-ui#590]
**Describe the bug**

Most times it works, but I've gotten a few PR's where the `macos` build failed and the JS code didn't appear to be the reason.

```
/usr/local/bin/npx cypress verify

25l[19:53:56]  Verifying Cypress can run /Users/runner/.cache/Cypress/6.1.0/Cypress.app [started]
[19:54:26]  Verifying Cypress can run /Users/runner/.cache/Cypress/6.1.0/Cypress.app [failed]
25hCypress verification timed out.

This command failed with the following output:

/Users/runner/.cache/Cypress/6.1.0/Cypress.app/Contents/MacOS/Cypress --no-sandbox --smoke-test --ping=747

----------

Command timed out after 30000 milliseconds: /Users/runner/.cache/Cypress/6.1.0/Cypress.app/Contents/MacOS/Cypress --no-sandbox --smoke-test --ping=747
Timed out

----------

Platform: darwin (19.6.0)
Cypress Version: 6.1.0
25h::debug::The process '/usr/local/bin/npx' failed with exit code 1
Error: The process '/usr/local/bin/npx' failed with exit code 1

```

It happened today for #586 . I did as before, and clicked the "Rerun all jobs" button and the second time it worked :man_shrugging: 

**Release version(s) and/or repository branch(es) affected?**

`master`

**Steps to reproduce the bug**

Trigger a workflow a few times, wait to see if the macos build passes or not.

**Expected behavior**

Consistent behavior.

**Screenshots**

**Additional context**

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by oliver-sanders** [comment]
May be fixed by upgrading to Yarn2 - https://github.com/cylc/cylc-ui/issues/593

**Comment by kinow** [comment]
@oliver-sanders what if we added `experimental: true` to matrix branch for macos?

![image](https://user-images.githubusercontent.com/304786/108267823-8bf29800-71d0-11eb-88e9-1abf2fc5a34e.png)

I think that would prevent the build from being marked as failure when the macos job failed? https://ncorti.com/blog/howto-github-actions-build-matrix#shadow-ci-jobs WDYT?

**Comment by oliver-sanders** [comment]
I guess could just comment it out of the test matrix for now since it's failing so reliably.

**Comment by kinow** [comment]
Unfortunately the build is failing again today :disappointed: I've committed to `master` disabling it again, re-opening this ticket too.

**Comment by MetRonnie** [comment]
Addressed by #1649 / superseded by #1528
